app/src/main/kotlin/maru/app/MaruAppCli.kt

```diff
@@ -17,6 +17,23 @@ import org.apache.logging.log4j.core.config.Configurator
 import picocli.CommandLine
 import picocli.CommandLine.Command
 
+internal class KebabToEnumConverter<T : Enum<T>>(
+  private val enumClass: Class<T>,
+) : CommandLine.ITypeConverter<T> {
+  override fun convert(value: String): T {
+    // Convert kebab-case to upper snake-case
+    val enumName = value.replace('-', '_').uppercase()
+    return try {
+      java.lang.Enum.valueOf(enumClass, enumName)
```

Comment:
Minor: Can you please add an import instead?

Reply:
I will leave it as is, cause import `java.lang.Enum` would conflict with the Kotlin `Enum` used in line 20